### ISSUE TITLE
did you mean to do "./cpanfile"?

### ISSUE THREAD

**Issue by anoadragon453** [matrix-org/sytest#442]
During the setup steps, I get the following while running `./install-deps`:

```
do "cpanfile" failed, '.' is no longer in @INC; did you mean do "./cpanfile"? at ./install-deps.pl line 61.
```

Not sure if do to env issue or the code itself. The following was put in my `.profile` by cpan:

```
PATH="/home/user/perl5/bin${PATH:+:${PATH}}"; export PATH;
PERL5LIB="/home/user/perl5/lib/perl5${PERL5LIB:+:${PERL5LIB}}"; export PERL5LIB;
PERL_LOCAL_LIB_ROOT="/home/user/perl5${PERL_LOCAL_LIB_ROOT:+:${PERL_LOCAL_LIB_ROOT}}"; export PERL_LOCAL_LIB_ROOT;
PERL_MB_OPT="--install_base \"/home/user/perl5\""; export PERL_MB_OPT;
PERL_MM_OPT="INSTALL_BASE=/home/user/perl5"; export PERL_MM_OPT;
```

**Comment by anoadragon453** [comment]
For reference, simple changing the last line to `do "./cpanfile"` worked.

**Comment by anoadragon453** [comment]
This was fixed in 2f7378cbe57ab0586524bc6873005de31db83909.
